bin/runtime-common/src/messages_benchmarking.rs

```diff
@@ -142,3 +145,47 @@ where
 			.expect("too many messages requested by benchmark"),
 	)
 }
+
+/// Prepare proof of messages delivery for the `receive_messages_delivery_proof` call.
+pub fn prepare_message_delivery_proof<B, H, R, ML, MH>(
+	params: MessageDeliveryProofParams<AccountIdOf<ThisChain<B>>>,
+	make_bridged_inbound_lane_data_key: ML,
+	make_bridged_header: MH,
+) -> FromBridgedChainMessagesDeliveryProof<B>
+where
+	B: MessageBridge,
+	H: Hasher,
+	R: pallet_substrate_bridge::Config,
+	<R::BridgedChain as bp_runtime::Chain>::Hash: Into<HashOf<BridgedChain<B>>>,
+	ML: Fn(LaneId) -> Vec<u8>,
+	MH: Fn(H::Out) -> <R::BridgedChain as bp_runtime::Chain>::Header,
+{
+	// prepare Bridged chain storage with inbound lane state
```

Comment:
Since out documentation is now using `source`/`target` chains we should also use that in our comments over `bridged`/`this` there possible. What do you think?

Reply:
It isn't always possible to say that chain A is the source chain and chain B is the target chain. Chain A may be the source of messages, but at the same time it is the target of the confirmations. So when we talk about confirmations, A is the target and B is the source. And even more, the same message lane module instance is used to send messages to the other chain and receive messages from the other chain. 

So I think there are places where we may use source/target chain terminology, but in the code there are much more places where we should refer the other chain as a  bridged chain.

Reply:
>  So when we talk about confirmations, A is the target and B is the source. And even more, the same message lane module instance is used to send messages to the other chain and receive messages from the other chain.

I agree with Hernando and would prefer having consistent naming. While I understand your point, about source/target being switched, I think we should not change the frame of reference here. The entire interaction is always `source -> target` chain and the frame of reference is message delivery, confirmations are simply delivered from `target chain` to `source chain` and I don't see an issue with this. Obviously from the relayer perspective "target" might be where you send transaction, but as I said we should not switch the frame of reference, but rather keep focused on consistently naming the chain in the "message passing" frame of reference.

That said, I do think it's totally fine to use "bridged chain" when it's clear from the context and secondly message lanes use bridge/this terminology all over, so it might actually be better to change this in a separate PR instead of randomly mixing the two.